Podfile

```diff
@@ -119,7 +115,7 @@ def networking_pods
   pod 'CocoaLumberjack', '~> 3.5'
   pod 'CocoaLumberjack/Swift', '~> 3.5'
 
-  pod 'Sourcery', '~> 1.0.3', :configuration => 'Debug'
+  pod 'Sourcery', '~> 1.0.3', configuration: 'Debug'
```

Comment:
Oh, now I'm curious to see what we use Sourcery for 🤓

Reply:
There's a dedicated Codegen target in the Xcode project if you are curious and want to take a look 😛